### DISCUSSION TITLE
WordPress plugin tested up to 6.2.2.  Timeline for 6.3?

### DISCUSSION THREAD

**Discussion by jcruz-ieeecs** [wp-graphql/wp-graphql#2913] [General] [answered]
Hello

WordPress 6.3 came out first week or so of August, and we're almost into Sept.    Any idea when the plugin will be released for v6.3?

Looking through the Github issues list and I've seen mention of WP 6.3, but was wondering if there was an update to this.

Thanks

**Comment by justlevine** [comment, marked as answer]
Hey @jcruz-ieeecs 

As of #2907 the repo is fully tested against WP 6.3, so you should be good to go; we just forget to update the plugin headers in the v1.15.0 release.

Afaik v1.14.x versions are also compatible will WP6.3 (inasmuch as no conflicts have been reported, and I personally have a bunch of production sites with no issues so far).

**Reply by jcruz-ieeecs** [reply]
Thanks, @justlevine.

We'll try the WP Plugin again.

When we attempted a few weeks ago, we noticed that the WP admin who installed the WP plugin could see the IDE and activated plugin. As soon as the other WP admins attempted to view the IDE, the plugin would deactivate itself.   

We tried to have one of the other WP admins install, just as I had done.   Sure enough, when I tried to view the IDE, he could no longer access, and the plugin was deactivated.

We'll try again, thanks.

**Reply by justlevine** [reply]
Tbh that sounds like a conflict with another plugin.

**Reply by jcruz-ieeecs** [reply]
Agreed.  We'll figure it out in our testing.

Thanks.
